project/plugins.sbt

```diff
@@ -1,2 +1,2 @@
-addSbtPlugin("com.typesafe.play" % "interplay" % "2.0.1")
-addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.3.15")
+addSbtPlugin("com.typesafe.play" % "interplay" % "2.0.4")
+addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.4.0-RC4")
```

Comment:
So that we can have a new release of play-doc using the same twirl version that Play 2.7.0 is using.